include/sot/core/madgwickahrs.hh

```diff
@@ -96,6 +96,9 @@ namespace dynamicgraph {
       void init(const double& dt);
       void set_beta(const double & beta);
 
+      /// Set the quaternion as [w,x,y,z]
+      void set_imu_quat(const dynamicgraph::Vector & imu_quat);
+
```

Comment:
The naming of the method is inconsistent across the class.
Here you low caps and line 114 you have major caps.
Here you are using _ and not in line 114.
It would be nice for the future to have a consistent convention.
Please chose one or the other. I personaly advocate to follow the 
Google convention.

Reply:
Yes, the whole class uses inconsistent conventions. I chose `set_imu_quat` because the signal is called `imu_quat`. For beta, it is `set_beta` in C++ and `setBeta` in Python, which is basically the opposite of what we should do.
So, what action should I take here? Do you want me to change this to `setImuQuat`? in C++? In Python? In both? But the class will still be inconsistent, I don't think I am going to change the other names...